### PR TITLE
update feature importance function

### PR DESCRIPTION
**Pull request by double-y**
### Description
* replace randomForest to ranger
* support numeric response column

### Checklist

Make sure you have performed following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [ ] Tested with Exploratory

### CODE REVIEW

File: R/randomForest_tidiers.R [hunk 1]
```diff
@@ -811,3 +834,142 @@ calc_feature_imp <- function(df,
...
+      ret <- data.frame(
+        actual_value = x$y,
+        predicted_value = x$predictions
+      ) %>%
+        dplyr::filter(!is.na(predicted_value))
+
+      if (is.numeric(ret$actual_value)){
+        tile <- cut(c(ret$actual_value, ret$predicted_value), 10)
+        ret$actual_bucket <- tile[seq(length(tile)/2)]
+        ret$predicted_bucket <- tile[length(tile)/2 + seq(length(tile)/2)]
```
Comment (hideaki) [inline R/randomForest_tidiers.R]: This conf_mat for regression part seems to be unfinished.
Please either make it work or remove.

File: NAMESPACE [hunk 2]
```diff
@@ -17,14 +17,17 @@ S3method(glance,randomForest.classification)
 S3method(glance,randomForest.formula)
 S3method(glance,randomForest.regression)
 S3method(glance,randomForest.unsupervised)
+S3method(glance,ranger)
 S3method(glance,xgb.Booster)
 S3method(tidy,bayesTest)
 S3method(tidy,bsts)
+S3method(tidy,fa)
```
Comment (hideaki) [inline NAMESPACE]: This should be removed till fa is in too?

File: R/randomForest_tidiers.R (hunk 1 above)
Comment (double-y) [inline R/randomForest_tidiers.R]: removed it

File: NAMESPACE (hunk 2 above)
Comment (double-y) [inline NAMESPACE]: removed it

**Pull request merged**